src/utils/fetch.js

```diff
@@ -1 +1,13 @@
-import axios from 'axios'
\ No newline at end of file
+import axios from 'axios'
+
```

Comment:
Please use axios.create() here so you will have a base URL which can be used for all your request functions. (See NC Notes/Axios bottom of the page)